compressor/tests/test_offline.py

```diff
@@ -72,10 +72,8 @@ def setUp(self):
         # TEMPLATES[1] to be Jinja2 templates backend in test_settings.
         TEMPLATES = copy.deepcopy(settings.TEMPLATES)
 
-        django_template_dir = os.path.join(
-            TEMPLATES[0]['DIRS'][0], self.templates_dir)
-        jinja2_template_dir = os.path.join(
-            TEMPLATES[1]['DIRS'][0], self.templates_dir)
+        django_template_dir = os.path.join(TEMPLATES[0]['DIRS'][0], self.templates_dir)
+        jinja2_template_dir = os.path.join(TEMPLATES[1]['DIRS'][0], self.templates_dir)
```

Comment:
I'm just using 99 chars limit, which is [allowed by PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). But I don't want to break your rules.